monai/transforms/inverse.py

```diff
@@ -76,8 +76,11 @@ def push_transform(
         info = {
             InverseKeys.CLASS_NAME: self.__class__.__name__,
             InverseKeys.ID: id(self),
-            InverseKeys.ORIG_SIZE: orig_size or (data[key].shape[1:] if hasattr(data[key], "shape") else None),
         }
+        if orig_size is not None:
+            InverseKeys.ORIG_SIZE = orig_size
+        elif hasattr(data[key], "shape"):
+            InverseKeys.ORIG_SIZE = data[key].shape[1:]
```

Comment:
missing the `else` here for the -1?

Reply:
sorry, i updated the docs. There's no reason to have a default (None or -1). If the object doesn't have `shape` then none of that batch will either. So no need to add anything. If shape doesn't exist in the forward direction, the orig_size won't be used in the inverse direction, either.

Reply:
...or that's what I thought until the CI/CD failed...